resources/form-schemas/configmap-ui-schema.json

```diff
@@ -10,37 +10,5 @@
   "immutable": {
```

Comment:
do we need to add a top level ui:title property with something like "ConfigMap Properties" ?